pkg/backupcontroller/cleanup.go

```diff
@@ -3,13 +3,66 @@ package backupcontroller
 import (
 	"context"
 	"fmt"
+	"os"
+	"strconv"
+	"strings"
 	"time"
 
 	"k8s.io/klog"
 
 	"kope.io/etcd-manager/pkg/backup"
 )
 
+// HourlyBackupsRetention controls how long hourly backups are kept.
+var HourlyBackupsRetention = 24 * 7 * time.Hour
+
+// DailyBackupsRetention controls how long daily backups are kept.
+var DailyBackupsRetention = 24 * 7 * 365 * time.Hour
+
+// ParseHumanDuration parses a go-style duration string, but
+// recognizes additional suffixes: d means "day" and is interpreted as
+// 24 hours; y means "year" and is interpreted as 365 days.
+func ParseHumanDuration(s string) (time.Duration, error) {
```

Comment:
This was shamelessly copied from `certs.go`. I couldn't find some good common utils place for it.
https://github.com/kopeio/etcd-manager/blob/84bd90884129b71ff5aae0aa36e88ab196dd2af5/pkg/pki/certs.go#L32